### PR TITLE
Fixing typos and minor suggestions

### PR DESCRIPTION
**Pull request by AlexJerabek**
Addressing some of the low-hanging fruit from the doc bash earlier today. This is mostly typos and grammar problems, but it also removes the use of the word "legacy", which may have negative implications.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[75f5efa](https://github.com/OfficeDev/office-scripts-docs/commits/75f5efa30ad9102cdb06ca06e3c126c5cdb01ce2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/excel-async-model.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/develop/excel-async-model.md) | :white_check_mark:Succeeded | |
[docs/develop/power-automate-integration.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/develop/power-automate-integration.md) | :white_check_mark:Succeeded | |
[docs/develop/scripting-fundamentals.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/develop/scripting-fundamentals.md) | :white_check_mark:Succeeded | |
[docs/includes/preview-note.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/includes/preview-note.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C8%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202007082151324580-143%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=t7sbeRTF8QUavu3WaI27vk2Sa7fuul%2F76RXuqDPrrPA%3D&st=2020-07-08T21%3A47%3A43Z&se=2020-08-08T21%3A52%3A43Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/develop/excel-async-model.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+ms.date: 07/08/2020
 localization_priority: Normal
 ---
 
 
-# Using the Office Scripts Async APIs to support legacy scripts
+# Support older Office Scripts that use the async APIs
 
-This article will teach you how to write scripts using the legacy, async, APIs. These APIs have the same core functionality as the standard, synchronous Office Scripts APIs, but they require that your script control the data synchronization between the script and the workbook.
+This article will teach you how to maintain and update scripts that use the older, async, APIs. These APIs have the same core functionality as the standard, synchronous Office Scripts APIs, but they require that your script control the data synchronization between the script and the workbook.
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/excel-async-model.md:11]:
```suggestion
This article will teach you how to maintain and update scripts that use the older model's async APIs. These APIs have the same core functionality as the now standard, synchronous Office Scripts APIs, but they require your script to control the data synchronization between the script and the workbook.
```

File: docs/develop/scripting-fundamentals.md [hunk 2]
```diff
@@ -13,18 +13,18 @@ This article will introduce you to the technical aspects of Office Scripts. You'
...
-// Your code goes here
+  // Your code goes here
 }
 ```
 
 The code inside the `main` function runs when the script is run. `main` can call other functions in your script, but code that's not contained in a function will not run.
 
 > [!CAUTION]
-> If your `main` function looks like `async function main(context: Excel.RequestContext)`, then your script is using the legacy, async API model. Please refer to [Using the Office Scripts Async APIs to support legacy scripts](excel-async-model.md) for more information, including how to convert your older script to the current API model.
+> If your `main` function looks like `async function main(context: Excel.RequestContext)`, then your script is using the older, async API model. Please refer to [Support older Office Scripts that use the Async APIs](excel-async-model.md) for more information, including how to convert your script to the current API model.
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/scripting-fundamentals.md:27]:
```suggestion
> If your `main` function looks like `async function main(context: Excel.RequestContext)`, your script is using the older async API model. For more information (including how to convert your script to the current API model), refer to [Support older Office Scripts that use the Async APIs](excel-async-model.md).
```

File: docs/develop/excel-async-model.md (hunk 1 above)
Comment (AlexJerabek) [reply docs/develop/excel-async-model.md:11]: Taking this suggestion with "now-standard" instead of "now standard".

**Comment by o365devx** [comment]
Docs Build status updates of commit _[cdbf467](https://github.com/OfficeDev/office-scripts-docs/commits/cdbf467bb7a2c21b07ac436801933c85bc3f10d3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/excel-async-model.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/develop/excel-async-model.md) | :white_check_mark:Succeeded | |
[docs/develop/power-automate-integration.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/develop/power-automate-integration.md) | :white_check_mark:Succeeded | |
[docs/develop/scripting-fundamentals.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/develop/scripting-fundamentals.md) | :white_check_mark:Succeeded | |
[docs/includes/preview-note.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/includes/preview-note.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C9%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202007091641102563-143%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=XyHnQIKIvFWwRn8sTSB%2BqfRQemt29M%2FEEThhadJrh%2Bw%3D&st=2020-07-09T16%3A37%3A27Z&se=2020-08-09T16%3A42%3A27Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[58d4bc3](https://github.com/OfficeDev/office-scripts-docs/commits/58d4bc366ba9f987f8f0949a12a52954b226321f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/excel-async-model.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/develop/excel-async-model.md) | :white_check_mark:Succeeded | |
[docs/develop/power-automate-integration.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/develop/power-automate-integration.md) | :white_check_mark:Succeeded | |
[docs/develop/scripting-fundamentals.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/develop/scripting-fundamentals.md) | :white_check_mark:Succeeded | |
[docs/includes/preview-note.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/includes/preview-note.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C9%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202007091641315252-143%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=JcMKHX3AWff4wZ%2FVGchCB39lXw8hEz6YPbFfuB3XOpk%3D&st=2020-07-09T16%3A38%3A43Z&se=2020-08-09T16%3A43%3A43Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f9874ed](https://github.com/OfficeDev/office-scripts-docs/commits/f9874edccc1f1d9bf5d16c5f3827e93dea719796)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/excel-async-model.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/develop/excel-async-model.md) | :white_check_mark:Succeeded | |
[docs/develop/power-automate-integration.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/develop/power-automate-integration.md) | :white_check_mark:Succeeded | |
[docs/develop/scripting-fundamentals.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/develop/scripting-fundamentals.md) | :white_check_mark:Succeeded | |
[docs/includes/preview-note.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/includes/preview-note.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-DocBashFirstRound/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C9%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202007091642356645-143%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=j8ePmwRfKEFELYZPdV%2FCZl3xBRLc57BeOI2bPryG1co%3D&st=2020-07-09T16%3A40%3A01Z&se=2020-08-09T16%3A45%3A01Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**